### PR TITLE
Version 0.5

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,20 +19,6 @@
...
-   * [Engine](https://github.com/openscilab/nava#engine)
-   * [Issues & Bug Reports](https://github.com/openscilab/nava#issues--bug-reports)
-   * [Contribution](https://github.com/openscilab/nava/blob/main/.github/CONTRIBUTING.md)
-   * [Authors](https://github.com/openscilab/nava/blob/main/AUTHORS.md)
-   * [License](https://github.com/openscilab/nava/blob/main/LICENSE)
-   * [Show Your Support](https://github.com/openscilab/nava#show-your-support)
-   * [Changelog](https://github.com/openscilab/nava/blob/main/CHANGELOG.md)
-   * [Code of Conduct](https://github.com/openscilab/nava/blob/main/.github/CODE_OF_CONDUCT.md)
-
-## Overview
```
Comment (sepandhaghighi) [inline README.md:35]: Why did you delete this line?

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sadrasabouri) [reply README.md:35]: I thought it’s redundant now. Since we are starting with overview anyways, there is no need to mention the title again.

Comment (sepandhaghighi) [reply README.md:35]: I suggest keeping the `## Overview` header to properly separate the badge and overview sections.

**Review by sepandhaghighi — APPROVED** [review]
LGTM 🔥
